docs/contrib/quickstart.rst

```diff
@@ -0,0 +1,123 @@
+.. _quickstart:
+
+Quick Start Guide
+=================
+
+Requirements
+------------
+
+- Java 8 or newer
+- sbt 1.5.8 or newer
+- LLVM/Clang 6.0 or newer
+
+Project Structure Overview
+--------------------------
+
+See :ref:`build`
+
+Project suffix
+--------------
+
+Most projects in ScalaNative cross-build against Scala ``2.12``, ``2.13`` and ``3``, and these projects have a suffix like ``2_12``, ``2_13`` or ``3`` to differentiate the Scala version.
+For example, ``sandbox`` has ``sandbox2_12``, ``sandbox2_13`` and ``sandbox3``.
+
+In the following we will use suffix ``3``, but remember that you can build and test for different versions using different suffixes.
+
+Commonn sbt commands
+--------------------
+
+**Common Test Commands**
+
+- ``tests3/test`` - run the unit tests for libraries on native build
+- ``tests3/testOnly org.scalanative.testsuite.javalib.util.RandomTest`` - run only the test of interest
+- ``tests3/testOnly *.RandomTest`` - run only the test of interest using wildcard
+- ``testsExt3/test`` - run the unit tests on native build, this module contains tests that requires dummy javalib implementation defined in ``javalibExtDummies``.
+- ``nir3/test`` - run the unit tests for NIR
+- ``tools3/test`` - run the unit tests of the tools: ScalaNative backend
```

Comment:
In case of nir running tests in native backend it's fine, but in case of tools the tests would now be skipped. It think it would be better to use `nirJVM3` and `toolsJVM3` here as this are the dominant targets

Reply:
Right, moved to `JVM` targets https://github.com/scala-native/scala-native/pull/3496/commits/b17677fcbae65b78f285eaf481f72076df2c81c7